vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -618,7 +665,30 @@ public Validator<LocalDateTime> getDefaultValidator() {
         return Validator.alwaysPass();
     }
 
+    @Override
+    public Registration addValidationStatusChangeListener(
+            ValidationStatusChangeListener<LocalDateTime> listener) {
+        return addClientValidatedEventListener(event -> listener
```

Comment:
This should be also behind the feature flag, similarly to DatePicker:

https://github.com/vaadin/flow-components/blob/6c9c142946dff768db7fb5763d8049d322daff73/vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java#L467-L478

It seems I missed that during the review of TimePicker, so it has a similar problem.